common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -571,3 +572,43 @@ def test_special_exams_menu_a11y(self):
             include=['section.edit-settings-timed-examination']
         )
         self.course_outline.a11y_audit.check_for_accessibility_errors()
+
+
+class StudioSettingsImageUploadTest(StudioCourseTest):
+    """
+    Class to test course settings image uploads.
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(StudioSettingsImageUploadTest, self).setUp()
+        self.settings_page = SettingsPage(self.browser, self.course_info['org'], self.course_info['number'],
+                                          self.course_info['run'])
+
+    @flaky(max_runs=3, min_passes=2)      # TODO fix this SOL-1776
+    def test_upload_course_card_image(self):
+        self.settings_page.visit()
+        self.settings_page.wait_for_page()
```

Comment:
@benpatterson pointed out that the above two lines are redundant. You only need to call `self.settings_page.visit()` here and in the other tests below.